samples-generator/lib/createBuildingsTile.js

```diff
@@ -91,24 +104,47 @@ function createBuildingsTile(options) {
         featureTableJson.RTC_CENTER = Cartesian3.pack(batchedMesh.getCenter(), new Array(3));
```

Comment:
`RTC_CENTER` is getting lost in the gltf/glb code path. If `--gltf` or `--glb` is set `createGltf` should create a node with a translation to replace the old `RTC_CENTER`/feature table behavior.

Reply:
fixed, see `injectTranslationNodeToGltfSpec.js` for details on how it works now. 

The batched examples always have a rotationMatrix as the root, so I create a translation node as a child of the rotationMatrix, then move the `mesh: 0` property from the root rotationMatrix to it's own node (as the child of the translationMatrix)